fluent-plugin-google-cloud.gemspec

```diff
@@ -10,7 +10,7 @@ eos
   gem.homepage      =
     'https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud'
   gem.license       = 'Apache-2.0'
-  gem.version       = '0.6.23'
+  gem.version       = '0.7.0'
```

Comment:
We have a bunch of pending changes. Do we want to wait until those are in before bumping the version?
Alternatively, should we bump the version off a clean release (so that pulling in fluentd 1.2 is the only change)? This would mean releasing 0.6.24 first.

Reply:
Yes, that makes perfect sense. I think we are releasing at `v0.6.24` next week. This will definitely wait till after that.